src/helpers.coffee

```diff
@@ -144,7 +147,13 @@ exports.calculateSpecificity = calculateSpecificity
 exports.isAtomModifier = (keystroke) ->
   AtomModifiers.has(keystroke) or AtomModifierRegex.test(keystroke)
 
-exports.keydownEvent = (key, {ctrl, shift, alt, cmd, keyCode, target, location}={}) ->
+exports.keydownEvent = (key, options) ->
+  return exports.keyboardEvent(key, 'keydown', options)
+
+exports.keyupEvent = (key, options) ->
+  return exports.keyboardEvent(key, 'keyup', options)
+
+exports.keyboardEvent = (key, eventType, {ctrl, shift, alt, cmd, keyCode, target, location}={}) ->
```

Comment:
Does this really need to be exported? Couldn't you use either of the two functions above it?